### PR TITLE
Update nodejs available versions

### PR DESCRIPTION
**Pull request by ludoge**
Switch documented nodejs versions to major versions only.
Don't provide a (non-exhaustive) list of deprecated available versions.

### CODE REVIEW

File: src/languages/nodejs.md [hunk 1]
```diff
@@ -4,22 +4,15 @@ Node.js is a popular JavaScript runtime built on Chrome's V8 JavaScript engine.
...
 
-The following versions are available but are not receiving security updates from upstream, so their use is not recommended. They will be removed at some point in the future.
-
-* 0.12
-* 4.7
-* 4.8
-* 6.9
-* 6.10
-* 8.2
+Some versions with a minor (such as 8.9) are available but are not receiving security updates from upstream, so their use is not recommended. They will be removed at some point in the future.
```
Comment (Crell) [inline src/languages/nodejs.md]: IMO we should still list the numbers as long as the containers themselves exist.

Comment (ludoge) [reply src/languages/nodejs.md]: That's a fair point, but the list there was nowhere near exhaustive. There are 20 available nodejs versions, I don't think we should list them all? A good compromise would be listing the latest deprecated minor (e.g. 8.11 but no 8.10 or 8.9), IMO

Comment (ralt) [reply src/languages/nodejs.md]: The list depends on the region. Do you really want to go down that rabbit hole? :)

Comment (Crell) [reply src/languages/nodejs.md]: Wait, we have 20 available node versions, or there are 20 node versions?  If we offered, why was that never documented?

If the former, then yeah, not listing all of them makes sense.

Comment (ralt) [reply src/languages/nodejs.md]: we have 20 available node versions

Comment (Crell) [reply src/languages/nodejs.md]: Dear lord...

**Pull request merged**